e2e/addons/hello/addons/secret.yml

```diff
@@ -0,0 +1,9 @@
+PasswordSecret:
```

Comment:
It might be good to mention that this is also something that's used in the test :D

Reply:
Done